### PR TITLE
std.algo.cmp.equal docu

### PR DESCRIPTION
**Pull request by burner**
* Params
* Returns

### CODE REVIEW

File: std/algorithm/comparison.d [hunk 1]
```diff
@@ -685,11 +685,18 @@ Compares two ranges for equality, as defined by predicate $(D pred)
...
-    $(D bool) for $(D a) in $(D r1) and $(D b) in $(D r2). Performs
-    $(BIGOH min(r1.length, r2.length)) evaluations of $(D pred).
+    This function compares to ranges for equality. The ranges may have
+    different element types, as long as $(D pred(a, b)) evaluates to $(D bool)
+    for $(D a) in $(D r1) and $(D b) in $(D r2). 
+    Performs $(BIGOH min(r1.length, r2.length)) evaluations of $(D pred).
+
+    Params:
+        r1 = The first part of the comparison.
+        r2 = The second part of the comparison.
```
Comment (quickfur) [inline std/algorithm/comparison.d]: Nitpick: a better wording would be: "r1 = The first range to be compared", "r2 = The second range to be compared".

**Comment by quickfur** [comment]
Other than that, LGTM. Thanks!

**Comment by quickfur** [comment]
Auto-merge toggled on

**Comment by burner** [comment]
thanks

File: std/algorithm/comparison.d [hunk 2]
```diff
@@ -685,11 +685,18 @@ Compares two ranges for equality, as defined by predicate $(D pred)
 template equal(alias pred = "a == b")
 {
     /++
-    Returns $(D true) if and only if the two ranges compare equal element
-    for element, according to binary predicate $(D pred). The ranges may
-    have different element types, as long as $(D pred(a, b)) evaluates to
-    $(D bool) for $(D a) in $(D r1) and $(D b) in $(D r2). Performs
-    $(BIGOH min(r1.length, r2.length)) evaluations of $(D pred).
+    This function compares to ranges for equality. The ranges may have
```
Comment (yazd) [inline std/algorithm/comparison.d]: s/to/two/

**Pull request merged**